dialogue-apache-hc5-client/src/main/java/com/palantir/dialogue/hc5/ApacheHttpClientBlockingChannel.java

```diff
@@ -339,6 +352,9 @@ public void close() {
                                             clientSnapshot.clientConfiguration().taggedMetricRegistry())
                                     .connectionClosedPartiallyConsumedResponse(clientSnapshot.name())
                                     .mark();
+                            // Do not call response.close which internally attempts to drain the response
```

Comment:
I didn't review the original PR, but runtime has #markConnectionNonReusable which maybe handles this whole business of closing things a bit more gracefully?

Reply:
Unfortunately it doesn’t help us solve the problem at hand because close still fully drains the response, regardless of the length.

Reply:
These docs are confusing me, maybe this is for a different version?

Reply:
Those are the 4.x docs